### PR TITLE
Add support for session id claim

### PR DESCRIPTION
**Pull request by lagr**

### CODE REVIEW

File: test/compeon/token/decoder_test.rb [hunk 1]
```diff
@@ -219,4 +219,14 @@ def test_with_an_invalid_iat_claim
...
+
+  def test_with_a_sid_claim
+    encoded_token = JWT.encode({ attr: 'Ein Attribut', knd: 'test', sid: '123456' }, PRIVATE_KEY, 'RS256')
+
+    Compeon::Token::Decoder.new(
+      encoded_token: encoded_token,
+      key: PRIVATE_KEY.public_key,
+      token_klass: TestToken
+    ).decode
+  end
```
Comment (David3141) [inline test/compeon/token/decoder_test.rb]: This test doesn't have any assertions, is it meant as smoke test?

Comment (lagr) [reply test/compeon/token/decoder_test.rb]: Yes, as there is no `refute_raises` this is how to assure no exception is raised.

**Review by David3141 — APPROVED** [review]
I'm not sure anymore what our conclusion was on this, do we need to adapt the `lib/compeon/access_token` as well?

**Comment by lagr** [comment]
@David3141 Might be a sensible thing to do, even though I believe `Compeon::AccessToken` is deprecated and was only still used in our `VerificationService`.

**Comment by lagr** [comment]
Revisiting the RFC, I think it would be correct not to put `sid` in `registered_claims_mapping`, as it is part of the customization at OpenID Connect Session Management and thus a custom claim.

**Pull request closed without merging**